### PR TITLE
add dtn7zero service and demo scenario

### PR DESCRIPTION
**Pull request by lh70**
Added dtn7zero as a dependency, a configurable dtn7zero service, and a demo scenario containing both dtn7zero and dtn7-rs.

### CODE REVIEW

File: parse_weather_data.py
Comment (gh0st42) [inline parse_weather_data.py:1]: the file should be renamed to something like `prepare_weather_data.py` as it does more than just parsing it.
as far as i can tell, it downloads data, unpacks it and converts it.

File: parse_weather_data.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    for filename in resources_to_parse:
+        csvs = fetch_csvs_from_url(f'{open_dwd_url}{filename}')
+
+        for csv_content in csvs:
+            temperatures += extract_temperature_list(csv_content)
+
+    os.makedirs(os.path.dirname(out_filepath))
+
+    with open(out_filepath, 'w') as f:
+        json.dump(temperatures, f)
```
Comment (gh0st42) [inline parse_weather_data.py]: why not work with CSVs directly? why use JSON for flat row data?

File: Dockerfile [hunk 2]
```diff
@@ -105,6 +105,19 @@ ENV DB_PATH="/app/moNNT.py/db.sqlite3" \
     NNTPPORT=1190 \
     NNTPSERVER=10.0.0.21
 
+
+# --------------------------- dtn7zero Installation ---------------------------
+RUN sudo pip3 install --upgrade requests && pip3 install dtn7zero==0.0.6 && sudo apt install python3-matplotlib
+
+RUN cd /app && git clone --recurse-submodules https://github.com/dtn7/dtn7zero.git && \
```
Comment (gh0st42) [inline Dockerfile]: why not install the whole lib here from the specified commit hash? the previous pip3 install might get out of sync and install a wrong version

**Review by gh0st42 — CHANGES_REQUESTED** [review]
Thank you for the PR.
some minor questions remain.

File: parse_weather_data.py (hunk 1 above)
Comment (lh70) [reply parse_weather_data.py]: the idea was to bring the data into exactly the format we need, so there is less overhead (loading+parsing) on demo-startup, and there is less code in the demo.

File: Dockerfile (hunk 2 above)
Comment (lh70) [reply Dockerfile]: The commit-hash is equivalent to the version packed & uploaded to PyPi, so there should not be any inconsistencies. The repository is cloned only to have access to the example-scripts. It is currently not used and could also just be removed from the showroom entirely.

Comment (lh70) [reply Dockerfile]: the unnecessary repository checkout is now removed entirely

File: parse_weather_data.py
Comment (lh70) [reply parse_weather_data.py:1]: the script is now renamed to prepare_weather_data.py

**Pull request merged**